docs/user-guide/inference-backend-management.md

```diff
@@ -0,0 +1,105 @@
+# Inference Backend Management
```

Comment:
Please check the latest change in https://github.com/gpustack/gpustack/pull/3455 for this